src/cpp/src/llm_pipeline_static.cpp

```diff
@@ -130,18 +132,19 @@ void merge_config_with(ov::AnyMap& lhs, const ov::AnyMap& rhs) {
     }
 }
 
-ov::AnyMap get_default_prefill_config() {
+ov::AnyMap get_default_prefill_config(const bool use_blobs = false) {
```

Comment:
maybe these functions shouldn't even know about `use_blobs` parameter, see the comments below how these cases could wrapped into separate functions

Reply:
Rewritten, thanks!